src/compiler/binder.ts

```diff
@@ -374,7 +364,7 @@ function getModuleInstanceStateWorker(node: Node, visited: Map<number, ModuleIns
             break;
         // 4. Export alias declarations pointing at only uninstantiated modules or things uninstantiated modules contain
         case SyntaxKind.ExportDeclaration:
-            const exportDeclaration = node as ExportDeclaration;
+            const exportDeclaration = node ;

```

Comment:
See this whitespace? This is almost certainly a bug in the eslint rule's `--fix` behavior. What's depressing is how many instances aren't caught by any of the whitespace-checking rules we have on, thoguh.

Reply:
@JoshuaKGoldberg my hero

Reply:
https://github.com/microsoft/TypeScript/pull/54148#discussion_r1186345078 🤝 ❤️